include/ocpp/common/pki_handler.hpp

```diff
@@ -3,7 +3,11 @@
 #ifndef OCPP_COMMON_PKI_HANDLER
 #define OCPP_COMMON_PKI_HANDLER
 
-#include <filesystem>
+#ifndef BOOSTFILESYSTEM
+<filesystem>
```

Comment:
Does this work without an `#include` before it?

Reply:
Thank you i fixed that.